### PR TITLE
feat(sdk): implementing `table.addRow`

### LINKED ISSUE #2274: cloud.Table Initial Rows
### Feature Spec

Allow users to populate entries into cloud.Table resource similar to how we do with `cloud.Bucket().put_object()`

### Use Cases

pre-populate table with data 

### Implementation Notes

_No response_

### Component

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by garysassano**
Closes #2274

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Comment by hasanaburayyan** [comment]
Hey @garysassano sorry for the delay I missed this one, going to review this now.

**Comment by hasanaburayyan** [comment]
@garysassano did you happen to check if this works on AWS? I get a unmarshal error.
```
│ Error: Invalid format of "item": Decoding failed: json: cannot unmarshal string into Go value of type dynamodb.AttributeValue
│ 
│   with aws_dynamodb_table_item.root_env0_cloudTable_DynamodbTableItemmario_0EBF907A,
│   on main.tf.json line 58, in resource.aws_dynamodb_table_item.root_env0_cloudTable_DynamodbTableItemmario_0EBF907A:
│   58:         "item": "{\"gender\":\"male\",\"role\":\"plumber\"}",
│ 
```

**Comment by garysassano** [comment]
@hasanaburayyan can you do another test?

**Comment by garysassano** [comment]
@hasanaburayyan Can you do another test? I think it should be fixed now.

File: libs/wingsdk/src/target-sim/table.inflight.ts [hunk 1]
```diff
@@ -13,16 +13,26 @@ export class Table implements ITableClient, ISimulatorResourceInstance {
...
     this.primaryKey = props.primaryKey;
     this.table = new Map<string, any>();
     this.context = context;
+    this.initialRows = props.initialRows ?? {};
   }
 
   public async init(): Promise<TableAttributes> {
+    for (const [key, row] of Object.entries(this.initialRows)) {
+      await this.context.withTrace({
+        message: `Adding row from preflight (key=${key}).`,
```
Comment (hasanaburayyan) [inline libs/wingsdk/src/target-sim/table.inflight.ts:30]: I think logging from `preflight` might be a little too technical. Maybe something like this
```suggestion
        message: `Adding initial row (key=${key}).`,
```

**Review by hasanaburayyan — APPROVED** [review]
Thanks for fixing the marshaling issue for AWS. 🎉 Left a minor comment feel free to resolve after addressed.

**Pull request merged**